### PR TITLE
Re-record cra/dist/index_chromium and migrate some tests

### PR DESCRIPTION
**Pull request by hbenl**
- the result counts in file-search-01 differ because the search has matches in the scripts our runtime injects into the page (for react and redux annotations) and those scripts are different between runtimes
- removed react_devtools-01-basic because it uses the Firefox recording and we have a corresponding test for Chromium
- migrating sourcemap_stacktrace is blocked on RUN-3135

### CODE REVIEW

**Comment by bvaughn** [comment]
Need to run `yarn lint`

File: packages/e2e-tests/tests/file-search-01.test.ts [hunk 1]
```diff
@@ -29,23 +29,23 @@ test("file-search-01: should search files", async ({ pageWithMeta: { page, recor
   // Verify search results for the string "test"
   await searchSources(page, "test");
   await verifySourceSearchOverflowMessageShown(page, false);
-  await verifySourceSearchSummary(page, "21 results in 1 file");
-  await verifyVisibleResultsCount(page, 25); // 21 results in 4 different files
+  await verifySourceSearchSummary(page, "12 results in 1 file");
+  await verifyVisibleResultsCount(page, 16); // 21 results in 4 different files
```
Comment (bvaughn) [inline packages/e2e-tests/tests/file-search-01.test.ts:33]: This comment is not correct anymore.

File: packages/e2e-tests/tests/file-search-01.test.ts [hunk 2]
```diff
@@ -29,23 +29,23 @@ test("file-search-01: should search files", async ({ pageWithMeta: { page, recor
...
-  await verifyVisibleResultsCount(page, 24);
+  await verifyVisibleResultsCount(page, 15);
 
   // Now include node_modules in the search
   await toggleExcludeNodeModulesCheckbox(page, false);
   await verifySourceSearchOverflowMessageShown(page, false);
-  await verifySourceSearchSummary(page, "25 results in 1 file");
-  await verifyVisibleResultsCount(page, 31); // 25 results in 6 different files
+  await verifySourceSearchSummary(page, "16 results in 1 file");
+  await verifyVisibleResultsCount(page, 22); // 25 results in 6 different files
```
Comment (bvaughn) [inline packages/e2e-tests/tests/file-search-01.test.ts:48]: Nor is this one.

**Review by bvaughn — CHANGES_REQUESTED** [review]
> removed react_devtools-01-basic because it uses the Firefox recording and we have a corresponding test for Chromium

That doesn't line up with the example key in the test you deleted:
```js
test.use({ exampleKey: "cra/dist/index_chromium.html" });
```

**Review by bvaughn — COMMENTED** [review]
I think the `packages/e2e-tests/tests/react_devtools-01-basic_chromium.test.ts` test is the one that uses the Firefox recording :(

File: packages/e2e-tests/tests/react_devtools-01-basic.test.ts [hunk 3]
```diff
@@ -1,116 +0,0 @@
...
-  jumpToMessageAndCheckComponents,
-  openReactDevtoolsPanel,
-  verifyInspectedPropertyValue,
-} from "../helpers/new-react-devtools-panel";
-import { getGetterValue } from "../helpers/object-inspector";
-import { clickScreenshot, hoverScreenshot } from "../helpers/screenshot";
-import { waitFor } from "../helpers/utils";
-import test, { expect } from "../testFixtureCloneRecording";
-
-test.use({ exampleKey: "cra/dist/index_chromium.html" });
```
Comment (bvaughn) [inline packages/e2e-tests/tests/react_devtools-01-basic.test.ts:23]: This test/file should not be deleted. The other one should.

**Comment by hbenl** [comment]
> I think the packages/e2e-tests/tests/react_devtools-01-basic_chromium.test.ts test is the one that uses the Firefox recording :(

You're right :laughing:
Will fix

File: packages/e2e-tests/tests/file-search-01.test.ts [hunk 4]
```diff
@@ -29,23 +29,23 @@ test("file-search-01: should search files", async ({ pageWithMeta: { page, recor
...
-  await verifyVisibleResultsCount(page, 24);
+  await verifyVisibleResultsCount(page, 15);
 
   // Now include node_modules in the search
   await toggleExcludeNodeModulesCheckbox(page, false);
   await verifySourceSearchOverflowMessageShown(page, false);
-  await verifySourceSearchSummary(page, "25 results in 1 file");
-  await verifyVisibleResultsCount(page, 31); // 25 results in 6 different files
+  await verifySourceSearchSummary(page, "16 results in 1 file");
+  await verifyVisibleResultsCount(page, 22); // 16 results in 6 different files
```
Comment (bvaughn) [inline packages/e2e-tests/tests/file-search-01.test.ts:48]: This comment is still wrong. Might as well delete it bc the assertion above has the correct summary/state.

File: packages/e2e-tests/tests/file-search-01.test.ts [hunk 5]
```diff
@@ -29,23 +29,23 @@ test("file-search-01: should search files", async ({ pageWithMeta: { page, recor
   // Verify search results for the string "test"
   await searchSources(page, "test");
   await verifySourceSearchOverflowMessageShown(page, false);
-  await verifySourceSearchSummary(page, "21 results in 1 file");
-  await verifyVisibleResultsCount(page, 25); // 21 results in 4 different files
+  await verifySourceSearchSummary(page, "12 results in 1 file");
+  await verifyVisibleResultsCount(page, 16); // 12 results in 4 different files
```
Comment (bvaughn) [inline packages/e2e-tests/tests/file-search-01.test.ts:33]: Something looks broken here. There are 12 results in 4 files, but the search summary says 12 results in 1 file.

Screenshot 2024-01-18 at 9 56 47 AM

**Review by bvaughn — APPROVED** [review]
I think we can land these changes but someone (I can do it) should dig into the search bug.

**Pull request merged**